src/ssl.c

```diff
@@ -29,6 +29,7 @@
     /* turn on GNU extensions for XVASPRINTF with wolfSSL_BIO_printf */
     #undef  _GNU_SOURCE
     #define _GNU_SOURCE
+    #include <stdarg.h>
```

Comment:
Can we avoid adding this for all builds and put into the user_settings.h or wrap with some other macro specific to fusion or put in the settings.h case for the fusion rtos define?

Reply:
Moved to user_settings.h